### DISCUSSION TITLE
Cannot enable Resize BAR on B365-D2V mainboard.

### DISCUSSION THREAD

**Discussion by btdzz** [xCuri0/ReBarUEFI#215] [Q&A]
I am using a B365 mainboard, along with an I5 9400F and RTX 3050 8GB. I have enabled Above 4G Decode, disabled CSM but still cannot enable Resize BAR. In the BIOS there is a Resizable BAR support section but it only has two options: off and auto. I selected auto but still cannot turn on ResizeBAR. GPUZ indicates that Resize BAR support is not enabled in the BIOS. Please help me enable Resize BAR. Thanks
![image](https://github.com/xCuri0/ReBarUEFI/assets/173114623/7fbbb729-51ec-45e1-b1b4-d1f0ef098273)

**Comment by xCuri0** [comment]
update your vbios, only reason for it to not work

**Reply by btdzz** [reply]
I searched and couldn't find any Vbios update for my RTX 3050 8GB. This is the latest and I updated it a while ago. I think the problem lies in the mainboard's BIOS.
![image](https://github.com/xCuri0/ReBarUEFI/assets/173114623/e264c148-cbe4-433a-91f3-dec41adafabd)

![image](https://github.com/xCuri0/ReBarUEFI/assets/173114623/595cdb20-b5b7-46c1-980a-398e201ab95b)

**Reply by xCuri0** [reply]
ask your mobo vendor about this, not here

**Reply by btdzz** [reply]
> ask your mobo vendor about this, not here
I changed to an AM4 mainboard. And, I think that error is because Intel simply hates users.
